dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -422,6 +405,41 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 						);
 					}
 
+					if (collection.collectionType === 'fixed/video') {
+						return (
+							<Section
```

Comment:
Sadly this carousel does not play well with FrontSection since it contains the left column and FrontSection does not have an option to go full width. Some of the padding can't be fitted in there easily either. 

But you are right about the show hide button, I will add this.

Reply:
Actually, I think it's a similar issue with show more in that there's no real space for it in when we use fullwidth so it will never be displayed. I think I'd like to address that in a separate PR if that's ok? Just to move this on a bit - we will need to make some design tweaks to this carousel anyway so I can address hiding as part of that branch, which I already have open.